README.md

```diff
@@ -31,7 +31,7 @@ MRTK3 is currently in **public preview.** We appreciate your feedback and contri
 
 ## Requirements
 
-MRTK3 requires Unity 2020.3.42, 2021.3.14 or higher. In addition, you need the [Mixed Reality Feature Tool for Unity](https://aka.ms/mrfeaturetool) to find, download, and add the packages to your project.
+MRTK3 requires Unity 2021.3.22f1 or higher. In addition, you need the [Mixed Reality Feature Tool for Unity](https://aka.ms/mrfeaturetool) to find, download, and add the packages to your project.
```

Comment:
2021.3.22f1 instead of 2021.3.21f1

Reply:
I initially tested with .22f1 before I found that .21f1 was the latest available on the build agents.  Now that .23f1 is the latest LTS, we should discuss what version to GA with.